### PR TITLE
Corrected and finished the debug option for the unified unattended installer

### PR DESCRIPTION
**Pull request by verdx**
|Related issue|
|---|
| #974, #839|

## Description
The PR does several changes:
 - The logger now prints its messages on the log file /var/log/wazuh-unattended-installer.log
 - The scripts `wazuh-cert-tool.sh` and `wazuh-password-tool.sh` have separate loggers when called individually but print to the general logger when called from the main function
 - The logger function is moved from `common.sh` to `unattended_installer.sh`
 - Removed `${debug}` from the `cd` commands that couldn't be removed

## Logs example

All-in-one installation shell output:
```
[root@centos7 unattended_scripts]# bash unattended_installation.sh -A -l
12/07/2021 09:42:36 INFO: Configuration file found. Creating certificates...
12/07/2021 09:42:37 INFO: Creating the Elasticsearch certificates...
12/07/2021 09:42:37 INFO: Creating Wazuh server certificates...
12/07/2021 09:42:37 INFO: Creating Kibana certificate...
12/07/2021 09:42:38 INFO: Certificates creation finished. They can be found in /vagrant/wazuh-packages/unattended_scripts/certs.
12/07/2021 09:42:38 INFO: Starting the installation...
12/07/2021 09:42:38 INFO: Starting the installation...
12/07/2021 09:42:38 INFO: Starting the installation...
12/07/2021 09:42:38 INFO: Installing all necessary utilities for the installation...
12/07/2021 09:42:38 INFO: Done
12/07/2021 09:42:38 INFO: Adding the Wazuh repository...
12/07/2021 09:42:38 INFO: Wazuh repository already exists skipping
12/07/2021 09:42:38 INFO: Done
12/07/2021 09:42:38 INFO: Installing the Wazuh manager...
12/07/2021 09:43:18 INFO: Done
12/07/2021 09:43:38 INFO: Wazuh-manager started
12/07/2021 09:43:38 INFO: Installing Open Distro for Elasticsearch...
12/07/2021 09:49:36 INFO: Done
12/07/2021 09:49:36 INFO: Configuring Elasticsearch...
12/07/2021 09:49:53 INFO: Elasticsearch started
12/07/2021 09:49:53 INFO: Initializing Elasticsearch...

12/07/2021 09:50:05 INFO: Done
12/07/2021 09:50:05 INFO: Installing Filebeat...
12/07/2021 09:50:32 INFO: Filebeat started
12/07/2021 09:50:32 INFO: Done
12/07/2021 09:50:32 INFO: Installing Open Distro for Kibana...
12/07/2021 09:59:25 INFO: Done
12/07/2021 09:59:54 INFO: Kibana started
```
Contents of `wazuh-unattended-installation.log`:
```
12/07/2021 09:37:24 WARNING: Installation cleaned. Check the /var/log/wazuh-unattended-installation.log file to learn more about the issue.
12/07/2021 09:41:51 INFO: Configuration file found. Creating certificates...
12/07/2021 09:41:51 INFO: Creating the Elasticsearch certificates...
12/07/2021 09:41:51 INFO: Creating Wazuh server certificates...
12/07/2021 09:41:51 INFO: Creating Kibana certificate...
12/07/2021 09:41:52 INFO: Certificates creation finished. They can be found in /vagrant/wazuh-packages/unattended_scripts/certs.
12/07/2021 09:41:52 INFO: Starting the installation...
12/07/2021 09:41:52 INFO: Starting the installation...
12/07/2021 09:41:52 INFO: Starting the installation...
12/07/2021 09:41:52 INFO: Installing all necessary utilities for the installation...
Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
 * base: mirror.librelabucm.org
 * extras: mirror.librelabucm.org
 * updates: mirror.librelabucm.org
Package curl-7.29.0-59.el7_9.1.x86_64 already installed and latest version
Package unzip-6.0-22.el7_9.x86_64 already installed and latest version
Package wget-1.14-18.el7_6.1.x86_64 already installed and latest version
Package libcap-2.22-11.el7.x86_64 already installed and latest version
Nothing to do
12/07/2021 09:41:52 INFO: Done
12/07/2021 09:41:52 INFO: Adding the Wazuh repository...
12/07/2021 09:41:52 INFO: Wazuh repository already exists skipping
12/07/2021 09:41:52 INFO: Done
12/07/2021 09:41:52 INFO: Installing the Wazuh manager...
Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
 * base: mirror.librelabucm.org
 * extras: mirror.librelabucm.org
 * updates: mirror.librelabucm.org
Resolving Dependencies
There are unfinished transactions remaining. You might consider running yum-complete-transaction, or "yum-complete-transaction --cleanup-only" and "yum history redo last", first to finish them. If those don't work you'll have to try removing/installing packages by hand (maybe package-cleanup can help).
--> Running transaction check
---> Package wazuh-manager.x86_64 0:4.2.5-1 will be installed
12/07/2021 09:41:54 ERROR: Wazuh installation failed
12/07/2021 09:41:54 WARNING: Cleaning the installation
12/07/2021 09:41:54 WARNING: Installation cleaned. Check the /var/log/wazuh-unattended-installation.log file to learn more about the issue.
12/07/2021 09:42:36 INFO: Configuration file found. Creating certificates...
mkdir: cannot create directory ‘/vagrant/wazuh-packages/unattended_scripts/certs’: File exists
12/07/2021 09:42:37 INFO: Creating the Elasticsearch certificates...
12/07/2021 09:42:37 INFO: Creating Wazuh server certificates...
12/07/2021 09:42:37 INFO: Creating Kibana certificate...
12/07/2021 09:42:38 INFO: Certificates creation finished. They can be found in /vagrant/wazuh-packages/unattended_scripts/certs.
12/07/2021 09:42:38 INFO: Starting the installation...
12/07/2021 09:42:38 INFO: Starting the installation...
12/07/2021 09:42:38 INFO: Starting the installation...
12/07/2021 09:42:38 INFO: Installing all necessary utilities for the installation...
Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
 * base: mirror.librelabucm.org
 * extras: mirror.librelabucm.org
 * updates: mirror.librelabucm.org
Package curl-7.29.0-59.el7_9.1.x86_64 already installed and latest version
Package unzip-6.0-22.el7_9.x86_64 already installed and latest version
Package wget-1.14-18.el7_6.1.x86_64 already installed and latest version
Package libcap-2.22-11.el7.x86_64 already installed and latest version
Nothing to do
12/07/2021 09:42:38 INFO: Done
12/07/2021 09:42:38 INFO: Adding the Wazuh repository...
12/07/2021 09:42:38 INFO: Wazuh repository already exists skipping
12/07/2021 09:42:38 INFO: Done
12/07/2021 09:42:38 INFO: Installing the Wazuh manager...
Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
 * base: mirror.librelabucm.org
 * extras: mirror.librelabucm.org
 * updates: mirror.librelabucm.org
Resolving Dependencies
There are unfinished transactions remaining. You might consider running yum-complete-transaction, or "yum-complete-transaction --cleanup-only" and "yum history redo last", first to finish them. If those don't work you'll have to try removing/installing packages by hand (maybe package-cleanup can help).
--> Running transaction check
---> Package wazuh-manager.x86_64 0:4.2.5-1 will be installed
--> Finished Dependency Resolution

Dependencies Resolved

================================================================================
 Package                Arch            Version            Repository      Size
================================================================================
Installing:
 wazuh-manager          x86_64          4.2.5-1            wazuh          111 M

Transaction Summary
================================================================================
Install  1 Package

Total size: 111 M
Installed size: 427 M
Downloading packages:
Running transaction check
Running transaction test
Transaction test succeeded
Running transaction
  Installing : wazuh-manager-4.2.5-1.x86_64                                 1/1 
  Verifying  : wazuh-manager-4.2.5-1.x86_64                                 1/1 

Installed:
  wazuh-manager.x86_64 0:4.2.5-1                                                
Complete!
12/07/2021 09:43:18 INFO: Done
Created symlink from /etc/systemd/system/multi-user.target.wants/wazuh-manager.service to /usr/lib/systemd/system/wazuh-manager.service.
12/07/2021 09:43:38 INFO: Wazuh-manager started
12/07/2021 09:43:38 INFO: Installing Open Distro for Elasticsearch...
Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
 * base: mirror.librelabucm.org
 * extras: mirror.librelabucm.org
 * updates: mirror.librelabucm.org
Resolving Dependencies
There are unfinished transactions remaining. You might consider running yum-complete-transaction, or "yum-complete-transaction --cleanup-only" and "yum history redo last", first to finish them. If those don't work you'll have to try removing/installing packages by hand (maybe package-cleanup can help).
--> Running transaction check
---> Package opendistroforelasticsearch.x86_64 0:1.13.2-1 will be installed
--> Processing Dependency: elasticsearch-oss = 7.10.2 for package: opendistroforelasticsearch-1.13.2-1.x86_64
--> Processing Dependency: opendistro-sql < 1.13.3.0 for package: opendistroforelasticsearch-1.13.2-1.x86_64
--> Processing Dependency: opendistro-security < 1.13.3.0 for package: opendistroforelasticsearch-1.13.2-1.x86_64
--> Processing Dependency: opendistro-reports-scheduler < 1.13.3.0 for package: opendistroforelasticsearch-1.13.2-1.x86_64
--> Processing Dependency: opendistro-performance-analyzer < 1.13.3.0 for package: opendistroforelasticsearch-1.13.2-1.x86_64
--> Processing Dependency: opendistro-knn < 1.13.3.0 for package: opendistroforelasticsearch-1.13.2-1.x86_64
--> Processing Dependency: opendistro-job-scheduler < 1.13.3.0 for package: opendistroforelasticsearch-1.13.2-1.x86_64
--> Processing Dependency: opendistro-index-management < 1.13.3.0 for package: opendistroforelasticsearch-1.13.2-1.x86_64
--> Processing Dependency: opendistro-asynchronous-search < 1.13.3.0 for package: opendistroforelasticsearch-1.13.2-1.x86_64
--> Processing Dependency: opendistro-anomaly-detection < 1.13.3.0 for package: opendistroforelasticsearch-1.13.2-1.x86_64
--> Processing Dependency: opendistro-alerting < 1.13.3.0 for package: opendistroforelasticsearch-1.13.2-1.x86_64
--> Processing Dependency: opendistro-sql >= 1.13.2.0 for package: opendistroforelasticsearch-1.13.2-1.x86_64
--> Processing Dependency: opendistro-security >= 1.13.1.0 for package: opendistroforelasticsearch-1.13.2-1.x86_64
--> Processing Dependency: opendistro-reports-scheduler >= 1.13.0.0 for package: opendistroforelasticsearch-1.13.2-1.x86_64
--> Processing Dependency: opendistro-performance-analyzer >= 1.13.0.0 for package: opendistroforelasticsearch-1.13.2-1.x86_64
--> Processing Dependency: opendistro-knn >= 1.13.0.0 for package: opendistroforelasticsearch-1.13.2-1.x86_64
--> Processing Dependency: opendistro-job-scheduler >= 1.13.0.0 for package: opendistroforelasticsearch-1.13.2-1.x86_64
--> Processing Dependency: opendistro-index-management >= 1.13.2.0 for package: opendistroforelasticsearch-1.13.2-1.x86_64
--> Processing Dependency: opendistro-asynchronous-search >= 1.13.0.1 for package: opendistroforelasticsearch-1.13.2-1.x86_64
--> Processing Dependency: opendistro-anomaly-detection >= 1.13.0.0 for package: opendistroforelasticsearch-1.13.2-1.x86_64
--> Processing Dependency: opendistro-alerting >= 1.13.1.0 for package: opendistroforelasticsearch-1.13.2-1.x86_64
--> Running transaction check
---> Package elasticsearch-oss.x86_64 0:7.10.2-1 will be installed
---> Package opendistro-alerting.noarch 0:1.13.1.0-1 will be installed
---> Package opendistro-anomaly-detection.noarch 0:1.13.0.0-1 will be installed
---> Package opendistro-asynchronous-search.noarch 0:1.13.0.1-1 will be installed
---> Package opendistro-index-management.noarch 0:1.13.2.0-1 will be installed
---> Package opendistro-job-scheduler.noarch 0:1.13.0.0-1 will be installed
---> Package opendistro-knn.noarch 0:1.13.0.0-1 will be installed
--> Processing Dependency: opendistro-knnlib = 1.13.0.0 for package: opendistro-knn-1.13.0.0-1.noarch
---> Package opendistro-performance-analyzer.noarch 0:1.13.0.0-1 will be installed
---> Package opendistro-reports-scheduler.noarch 0:1.13.0.0-1 will be installed
---> Package opendistro-security.noarch 0:1.13.1.0-1 will be installed
---> Package opendistro-sql.noarch 0:1.13.2.0-1 will be installed
--> Running transaction check
---> Package opendistro-knnlib.x86_64 0:1.13.0.0-1 will be installed
--> Finished Dependency Resolution

Dependencies Resolved

================================================================================
 Package                             Arch       Version         Repository
                                                                           Size
================================================================================
Installing:
 opendistroforelasticsearch          x86_64     1.13.2-1        wazuh     3.2 k
Installing for dependencies:
 elasticsearch-oss                   x86_64     7.10.2-1        wazuh     220 M
 opendistro-alerting                 noarch     1.13.1.0-1      wazuh      13 M
 opendistro-anomaly-detection        noarch     1.13.0.0-1      wazuh     8.7 M
 opendistro-asynchronous-search      noarch     1.13.0.1-1      wazuh     166 k
 opendistro-index-management         noarch     1.13.2.0-1      wazuh     6.9 M
 opendistro-job-scheduler            noarch     1.13.0.0-1      wazuh     955 k
 opendistro-knn                      noarch     1.13.0.0-1      wazuh     2.7 M
 opendistro-knnlib                   x86_64     1.13.0.0-1      wazuh     545 k
 opendistro-performance-analyzer     noarch     1.13.0.0-1      wazuh      62 M
 opendistro-reports-scheduler        noarch     1.13.0.0-1      wazuh     5.2 M
 opendistro-security                 noarch     1.13.1.0-1      wazuh      38 M
 opendistro-sql                      noarch     1.13.2.0-1      wazuh      15 M

Transaction Summary
================================================================================
Install  1 Package (+12 Dependent packages)

Total download size: 373 M
Installed size: 571 M
Downloading packages:
--------------------------------------------------------------------------------
Total                                              1.1 MB/s | 373 MB  05:44     
Running transaction check
Running transaction test
Transaction test succeeded
Running transaction
Creating elasticsearch group... OK
Creating elasticsearch user... OK
  Installing : elasticsearch-oss-7.10.2-1.x86_64                           1/13 
### NOT starting on installation, please execute the following statements to configure elasticsearch service to start automatically using systemd
 sudo systemctl daemon-reload
 sudo systemctl enable elasticsearch.service
### You can start elasticsearch service by executing
 sudo systemctl start elasticsearch.service
  Installing : opendistro-index-management-1.13.2.0-1.noarch               2/13 
  Installing : opendistro-alerting-1.13.1.0-1.noarch                       3/13 
  Installing : opendistro-performance-analyzer-1.13.0.0-1.noarch           4/13 
# Enabling opendistro performance analyzer to start and stop along with elasticsearch.service
Created symlink from /etc/systemd/system/elasticsearch.service.wants/opendistro-performance-analyzer.service to /usr/lib/systemd/system/opendistro-performance-analyzer.service.
  Installing : opendistro-sql-1.13.2.0-1.noarch                            5/13 
  Installing : opendistro-asynchronous-search-1.13.0.1-1.noarch            6/13 
  Installing : opendistro-anomaly-detection-1.13.0.0-1.noarch              7/13 
  Installing : opendistro-security-1.13.1.0-1.noarch                       8/13 
OpenDistro for Elasticsearch Security Demo Installer
 ** Warning: Do not use on production or public reachable systems **
Basedir: /usr/share/elasticsearch
This script maybe require your root password for 'sudo' privileges
Elasticsearch install type: rpm/deb on CentOS Linux release 7.8.2003 (Core)
Elasticsearch config dir: /etc/elasticsearch
Elasticsearch config file: /etc/elasticsearch/elasticsearch.yml
Elasticsearch bin dir: /usr/share/elasticsearch/bin
Elasticsearch plugins dir: /usr/share/elasticsearch/plugins
Elasticsearch lib dir: /usr/share/elasticsearch/lib
Detected Elasticsearch Version: x-content-7.10.2
Detected Open Distro Security Version: 1.13.1.0

### Success
### Execute this script now on all your nodes and then start all nodes
### Open Distro Security will be automatically initialized.
### If you like to change the runtime configuration 
### change the files in ../securityconfig and execute: 
sudo "/usr/share/elasticsearch/plugins/opendistro_security/tools/securityadmin.sh" -cd "/usr/share/elasticsearch/plugins/opendistro_security/securityconfig" -icl -key "/etc/elasticsearch/kirk-key.pem" -cert "/etc/elasticsearch/kirk.pem" -cacert "/etc/elasticsearch/root-ca.pem" -nhnv
### or run ./securityadmin_demo.sh
### To use the Security Plugin ConfigurationGUI
### To access your secured cluster open https://<hostname>:<HTTP port> and log in with admin/admin.
### (Ignore the SSL certificate warning because we installed self-signed demo certificates)
  Installing : opendistro-reports-scheduler-1.13.0.0-1.noarch              9/13 
  Installing : opendistro-job-scheduler-1.13.0.0-1.noarch                 10/13 
  Installing : opendistro-knnlib-1.13.0.0-1.x86_64                        11/13 
  Installing : opendistro-knn-1.13.0.0-1.noarch                           12/13 
  Installing : opendistroforelasticsearch-1.13.2-1.x86_64                 13/13 
Created elasticsearch keystore in /etc/elasticsearch/elasticsearch.keystore
  Verifying  : opendistro-index-management-1.13.2.0-1.noarch               1/13 
  Verifying  : opendistro-knn-1.13.0.0-1.noarch                            2/13 
  Verifying  : opendistro-knnlib-1.13.0.0-1.x86_64                         3/13 
  Verifying  : opendistro-alerting-1.13.1.0-1.noarch                       4/13 
  Verifying  : opendistro-performance-analyzer-1.13.0.0-1.noarch           5/13 
  Verifying  : opendistro-sql-1.13.2.0-1.noarch                            6/13 
  Verifying  : elasticsearch-oss-7.10.2-1.x86_64                           7/13 
  Verifying  : opendistroforelasticsearch-1.13.2-1.x86_64                  8/13 
  Verifying  : opendistro-asynchronous-search-1.13.0.1-1.noarch            9/13 
  Verifying  : opendistro-anomaly-detection-1.13.0.0-1.noarch             10/13 
  Verifying  : opendistro-security-1.13.1.0-1.noarch                      11/13 
  Verifying  : opendistro-reports-scheduler-1.13.0.0-1.noarch             12/13 
  Verifying  : opendistro-job-scheduler-1.13.0.0-1.noarch                 13/13 

Installed:
  opendistroforelasticsearch.x86_64 0:1.13.2-1                                  

Dependency Installed:
  elasticsearch-oss.x86_64 0:7.10.2-1                                           
  opendistro-alerting.noarch 0:1.13.1.0-1                                       
  opendistro-anomaly-detection.noarch 0:1.13.0.0-1                              
  opendistro-asynchronous-search.noarch 0:1.13.0.1-1                            
  opendistro-index-management.noarch 0:1.13.2.0-1                               
  opendistro-job-scheduler.noarch 0:1.13.0.0-1                                  
  opendistro-knn.noarch 0:1.13.0.0-1                                            
  opendistro-knnlib.x86_64 0:1.13.0.0-1                                         
  opendistro-performance-analyzer.noarch 0:1.13.0.0-1                           
  opendistro-reports-scheduler.noarch 0:1.13.0.0-1                              
  opendistro-security.noarch 0:1.13.1.0-1                                       
  opendistro-sql.noarch 0:1.13.2.0-1                                            

Complete!
12/07/2021 09:49:36 INFO: Done
12/07/2021 09:49:36 INFO: Configuring Elasticsearch...
-> removing [opendistro-performance-analyzer]...
Created symlink from /etc/systemd/system/multi-user.target.wants/elasticsearch.service to /usr/lib/systemd/system/elasticsearch.service.
12/07/2021 09:49:53 INFO: Elasticsearch started
12/07/2021 09:49:53 INFO: Initializing Elasticsearch...
Open Distro Security Admin v7
Will connect to localhost:9300 ... done
Connected as CN=admin,OU=Docu,O=Wazuh,L=California,C=US
Elasticsearch Version: 7.10.2
Open Distro Security Version: 1.13.1.0
Contacting elasticsearch cluster 'elasticsearch' and wait for YELLOW clusterstate ...
Clustername: wazuh-cluster
Clusterstate: GREEN
Number of nodes: 1
Number of data nodes: 1
.opendistro_security index does not exists, attempt to create it ... done (0-all replicas)
Populate config from /usr/share/elasticsearch/plugin/opendistro_security/securityconfig/
Will update '_doc/config' with /usr/share/elasticsearch/plugins/opendistro_security/securityconfig/config.yml 
   SUCC: Configuration for 'config' created or updated
Will update '_doc/roles' with /usr/share/elasticsearch/plugins/opendistro_security/securityconfig/roles.yml 
   SUCC: Configuration for 'roles' created or updated
Will update '_doc/rolesmapping' with /usr/share/elasticsearch/plugins/opendistro_security/securityconfig/roles_mapping.yml 
   SUCC: Configuration for 'rolesmapping' created or updated
Will update '_doc/internalusers' with /usr/share/elasticsearch/plugins/opendistro_security/securityconfig/internal_users.yml 
   SUCC: Configuration for 'internalusers' created or updated
Will update '_doc/actiongroups' with /usr/share/elasticsearch/plugins/opendistro_security/securityconfig/action_groups.yml 
   SUCC: Configuration for 'actiongroups' created or updated
Will update '_doc/tenants' with /usr/share/elasticsearch/plugins/opendistro_security/securityconfig/tenants.yml 
   SUCC: Configuration for 'tenants' created or updated
Will update '_doc/nodesdn' with /usr/share/elasticsearch/plugins/opendistro_security/securityconfig/nodes_dn.yml 
   SUCC: Configuration for 'nodesdn' created or updated
Will update '_doc/whitelist' with /usr/share/elasticsearch/plugins/opendistro_security/securityconfig/whitelist.yml 
   SUCC: Configuration for 'whitelist' created or updated
Will update '_doc/audit' with /usr/share/elasticsearch/plugins/opendistro_security/securityconfig/audit.yml 
   SUCC: Configuration for 'audit' created or updated
Done with success
12/07/2021 09:50:05 INFO: Done
12/07/2021 09:50:05 INFO: Installing Filebeat...
There are unfinished transactions remaining. You might consider running yum-complete-transaction, or "yum-complete-transaction --cleanup-only" and "yum history redo last", first to finish them. If those don't work you'll have to try removing/installing packages by hand (maybe package-cleanup can help).
wazuh/
wazuh/module.yml
wazuh/archives/
wazuh/archives/config/
wazuh/archives/config/archives.yml
wazuh/archives/ingest/
wazuh/archives/ingest/pipeline.json
wazuh/archives/manifest.yml
wazuh/alerts/
wazuh/alerts/config/
wazuh/alerts/config/alerts.yml
wazuh/alerts/ingest/
wazuh/alerts/ingest/pipeline.json
wazuh/alerts/manifest.yml
wazuh/_meta/
wazuh/_meta/config.yml
wazuh/_meta/fields.yml
wazuh/_meta/docs.asciidoc
Created symlink from /etc/systemd/system/multi-user.target.wants/filebeat.service to /usr/lib/systemd/system/filebeat.service.
12/07/2021 09:50:32 INFO: Filebeat started
12/07/2021 09:50:32 INFO: Done
12/07/2021 09:50:32 INFO: Installing Open Distro for Kibana...
Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
 * base: mirror.librelabucm.org
 * extras: mirror.librelabucm.org
 * updates: mirror.librelabucm.org
Resolving Dependencies
There are unfinished transactions remaining. You might consider running yum-complete-transaction, or "yum-complete-transaction --cleanup-only" and "yum history redo last", first to finish them. If those don't work you'll have to try removing/installing packages by hand (maybe package-cleanup can help).
--> Running transaction check
---> Package opendistroforelasticsearch-kibana.x86_64 0:1.13.2-1 will be installed
--> Finished Dependency Resolution

Dependencies Resolved

================================================================================
 Package                               Arch       Version       Repository
                                                                           Size
================================================================================
Installing:
 opendistroforelasticsearch-kibana     x86_64     1.13.2-1      wazuh     224 M

Transaction Summary
================================================================================
Install  1 Package

Total download size: 224 M
Installed size: 660 M
Downloading packages:
Running transaction check
Running transaction test
Transaction test succeeded
Running transaction
  Installing : opendistroforelasticsearch-kibana-1.13.2-1.x86_64            1/1 
chown: cannot access ‘/usr/share/kibana/optimize’: No such file or directory
no optimize folder
  Verifying  : opendistroforelasticsearch-kibana-1.13.2-1.x86_64            1/1 

Installed:
  opendistroforelasticsearch-kibana.x86_64 0:1.13.2-1                           

Complete!
12/07/2021 09:59:25 INFO: Done
Attempting to transfer from https://packages.wazuh.com/4.x/ui/kibana/wazuh_kibana-4.2.5_7.10.2-1.zip
Transferring 32888474 bytes....................
Transfer complete
Retrieving metadata from plugin archive
Extracting plugin archive
Extraction complete
Plugin installation complete
Created symlink from /etc/systemd/system/multi-user.target.wants/kibana.service to /etc/systemd/system/kibana.service.
12/07/2021 09:59:54 INFO: Kibana started
```

## Tests
- All-in-one installation:
  - [x] CentOS 7 
  - [x] Debian 9
- Distributed
  - [x] CentOS 7
  - [x] Debian 9

### CODE REVIEW

**Review by lastranikos — COMMENTED** [review]
Is it necessary to have multiple logger functions?
and with different logfiles?

logger() {
logger_cert() {
logger_pass() {

File: unattended_scripts/install_functions/opendistro/wazuh-cert-tool.sh [hunk 1]
```diff
@@ -8,7 +8,10 @@
 # License (version 2) as published by the FSF - Free Software
 # Foundation.
 
-debug='> /dev/null 2>&1'
+if [[ -z "${logfile}" ]]; then
+    logfile="/var/log/wazuh-cert-tool.log"
```
[the remark this answers is not in the document]
Comment (lastranikos) [reply unattended_scripts/install_functions/opendistro/wazuh-cert-tool.sh:12]: cool, LGTM

**Review by lastranikos — APPROVED** [review]
LGTM

**Review by rauldpm — CHANGES_REQUESTED** [review]
Good job, jsut a minor change. Review calls to echo, it must use logger

https://github.com/wazuh/wazuh-packages/blob/3af38121086e8f1d9e063762d31e6248e4f64945/unattended_scripts/install_functions/opendistro/elasticsearch.sh#L14

**Comment by verdx** [comment]
> Good job, just a minor change. Review calls to echo, it must use logger
> 
> https://github.com/wazuh/wazuh-packages/blob/3af38121086e8f1d9e063762d31e6248e4f64945/unattended_scripts/install_functions/opendistro/elasticsearch.sh#L14

Changed in f97aeaa032877edde2aba5b649934742b24033a7

**Review by rauldpm — APPROVED** [review]
LGTM

**Pull request merged**